### ISSUE TITLE
Documentation example for `introspect` fails

### ISSUE THREAD

**Issue by Hydrotoast** [dcjones/Compose.jl#215]
Hi,

I ran the documentation example for using `introspect` and a `MethodError` was thrown.

```julia
julia> tomato_bisque =
           compose(context(),
                   (context(), circle(), fill("bisque")),
                   (context(), rectangle(), fill("tomato")))
Compose.Context(Measures.BoundingBox{Tuple{Measures.Length{:w,Float64},Measures.Length{:h,Float64}},Tuple{Measures.Length{:w,Float64},Measures.Length{:h,Float64}}}((0.0w,0.0h),(1.0w,1.0h)),Nullable{Compose.UnitBox{S,T,U,V}}(),Nullable{Compose.Rotation{P<:NTuple{N,Measures.Measure}}}(),Nullable{Compose.Mirror}(),List([Compose.Context(Measures.BoundingBox{Tuple{Measures.Length{:w,Float64},Measures.Length{:h,Float64}},Tuple{Measures.Length{:w,Float64},Measures.Length{:h,Float64}}}((0.0w,0.0h),(1.0w,1.0h)),Nullable{Compose.UnitBox{S,T,U,V}}(),Nullable{Compose.Rotation{P<:NTuple{N,Measures.Measure}}}(),Nullable{Compose.Mirror}(),List([]),List([Compose.Form{Compose.RectanglePrimitive{Tuple{Measures.Length{:w,Float64},Measures.Length{:h,Float64}},Measures.Length{:w,Float64},Measures.Length{:h,Float64}}}([Compose.RectanglePrimitive{Tuple{Measures.Length{:w,Float64},Measures.Length{:h,Float64}},Measures.Length{:w,Float64},Measures.Length{:h,Float64}}((0.0w,0.0h),1.0w,1.0h)],symbol(""))]),List([Compose.Property{Compose.FillPrimitive}([Compose.FillPrimitive(RGBA{Float64}(1.0,0.38823529411764707,0.2784313725490196,1.0))])]),0,false,false,false,false,nothing,nothing,0.0,symbol("")), Compose.Context(Measures.BoundingBox{Tuple{Measures.Length{:w,Float64},Measures.Length{:h,Float64}},Tuple{Measures.Length{:w,Float64},Measures.Length{:h,Float64}}}((0.0w,0.0h),(1.0w,1.0h)),Nullable{Compose.UnitBox{S,T,U,V}}(),Nullable{Compose.Rotation{P<:NTuple{N,Measures.Measure}}}(),Nullable{Compose.Mirror}(),List([]),List([Compose.Form{Compose.CirclePrimitive{Tuple{Measures.Length{:w,Float64},Measures.Length{:h,Float64}},Measures.Length{:w,Float64}}}([Compose.CirclePrimitive{Tuple{Measures.Length{:w,Float64},Measures.Length{:h,Float64}},Measures.Length{:w,Float64}}((0.5w,0.5h),0.5w)],symbol(""))]),List([Compose.Property{Compose.FillPrimitive}([Compose.FillPrimitive(RGBA{Float64}(1.0,0.8941176470588235,0.7686274509803921,1.0))])]),0,false,false,false,false,nothing,nothing,0.0,symbol(""))]),List([]),List([]),0,false,false,false,false,nothing,nothing,0.0,symbol(""))

julia> introspect(tomato_bisque)
ERROR: MethodError: `convert` has no method matching convert(::Type{Measures.Length{:cx,Float64}}, ::Measures.Add{Measures.Length{:cx,Float64},Measures.Length{:mm,Float64}})
This may have arisen from a call to the constructor Measures.Length{:cx,Float64}(...),
since type constructors fall back to convert methods.
Closest candidates are:
  Measures.Length{U,T}(::Any)
  call{T}(::Type{T}, ::Any)
  convert{u,T1<:Number,T2<:Number}(::Type{Measures.Length{u,T1<:Number}}, ::Measures.Length{u,T2<:Number})
  ...
 in setindex! at array.jl:314
 in polygon at /home/gcielo/.julia/v0.4/Compose/src/form.jl:76
 in polygon at /home/gcielo/.julia/v0.4/Compose/src/form.jl:67
 in introspect at /home/gcielo/.julia/v0.4/Compose/src/container.jl:651
```

**Comment by Hydrotoast** [comment]
Seems to be related to #191.

**Comment by Hydrotoast** [comment]
Fixed by #191
